css/style.css

```diff
@@ -65,8 +65,8 @@ section {
 }
 
 .image > img {
-    width: 350px;
-    height: 350px;
+    width: 300px;
+    height: 300px;
 }
 
```

Comment:
Good work, though I would carefuly consider the circumstances before using absolute units of length